.gitignore

```diff
@@ -13,3 +13,6 @@ hs_err_pid*
 .idea/
 target/
 *.iml
+
+testng
+testng1.xml
```

Comment:
do you sure u need to exclude this file?

Reply:
fixed